app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from .cafe import Cafe
+from .errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
+    missing_masks = 0
+    for friend in friends:
```

Comment:
I suggest separate the loop from two sides with an empty line for better readability